tests/kolainst/nondestructive/misc.sh

```diff
@@ -4,6 +4,15 @@ set -xeuo pipefail
 . ${KOLA_EXT_DATA}/libtest.sh
 cd $(mktemp -d)
 
+# Ensure multicall is correctly set up and working.
+if ! test $(which rpm-ostree) -ef $(which ostree-container); then
```

Comment:
This "same inode" test seems to fails on CI.
I'm unsure about why, but I think it's a side-effect of how we inject custom files into the image under test.
While slightly unfortunate, I'm going to convert this into a "same digest" check instead.